proto/anki/import_export.proto

```diff
@@ -45,8 +47,21 @@ message ExportCollectionPackageRequest {
   bool legacy = 3;
 }
 
+message ImportAnkiPackageOptions {
+  enum UpdateCondition {
```

Comment:
Minor and not a required change. From https://protobuf.dev/programming-guides/style/:

>  Prefer prefixing enum values instead of surrounding them in an enclosing message.

I used to think this was silly ("why should I have to repeat the prefix each time just because C++ is bad?). But I've come around to it, because the nesting approach is rather awkward, especially in Rust. If you have two Foo enums nested in messages Bar and Baz, you have to import bar::Foo and baz::Foo, and it gets confusing what you're referring to. If instead you define the enums at the top level and give them a suitable name (eg BarFoo and BazFoo), it's immediately obvious which one you're referring to, and you don't need to deal with imports from submodules.  There's no longer a need to deal with multiple enums with the same name. I've started to feel the same way about other nested messages as well - keeping them at the top level and giving them a more specific name tends to make things easier (eg ImportAnkiPackageUpdateCondition). The prefixes it requires can end up rather hideous though unfortunately. I wish there was a decent RPC scheme that had the language support Protobuf does, but had better first-class support for things like discriminated unions.